src/test/resources/pages/instructorSearchPageSearchFeedbackResponseComments.html

```diff
@@ -216,4 +216,4 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
@whipermr5 there's a reason I did not; you can give it a try yourself.
(of course, you can have another lintspaces config for HTML test files)